### PR TITLE
feat(metadata-view): Add name field by default to md query

### PR DESCRIPTION
**Pull request by manishPh**

### CODE REVIEW

File: src/features/metadata-based-view/MetadataQueryAPIHelper.js [hunk 1]
```diff
@@ -222,4 +226,24 @@ export default class MetadataQueryAPIHelper {
             .getMetadataAPI(true)
             .updateMetadata(file, this.metadataTemplate, operations, successsCallback, errorCallback);
     };
+
+    /**
+     * Verify that the metadata query has reqiured fields and update it if necessary
```
Comment (swfree) [inline src/features/metadata-based-view/MetadataQueryAPIHelper.js:231]: minor typo: required

File: src/features/metadata-based-view/MetadataQueryAPIHelper.js [hunk 2]
```diff
@@ -222,4 +226,24 @@ export default class MetadataQueryAPIHelper {
...
+     * For a file item, default fields included in the response are "type", "id", "etag"
+     *
+     * @param {Object} metadataQuery metadata query object
```
Comment (swfree) [inline src/features/metadata-based-view/MetadataQueryAPIHelper.js:234]: should `{Object}` be `{MetadataQueryType}` here & in the line below?

File: src/features/metadata-based-view/MetadataQueryAPIHelper.js [hunk 3]
```diff
@@ -38,6 +40,8 @@ const SELECT_TYPES: Array<typeof METADATA_FIELD_TYPE_ENUM | typeof METADATA_FIEL
     METADATA_FIELD_TYPE_MULTISELECT,
 ];
 
+const METADATA_QUERY_ITEM_NAME_FIELD = 'name';
```
Comment (sahiga) [inline src/features/metadata-based-view/MetadataQueryAPIHelper.js:43]: Can you use `FIELD_NAME` from constants.js here?

File: src/features/metadata-based-view/__tests__/MetadataQueryAPIHelper.test.js [hunk 4]
```diff
@@ -421,4 +424,50 @@ describe('features/metadata-based-view/MetadataQueryAPIHelper', () => {
...
+                expect(isArray(updatedMetadataQuery.fields)).toBe(true);
+                expect(includes(updatedMetadataQuery.fields, 'name')).toBe(true);
+
+                if (index === 2) {
+                    // Verify "name" is added to pre-existing fields
+                    expect(updatedMetadataQuery.fields).toEqual([...mdQueryWithoutNameField.fields, 'name']);
+                }
+
+                if (index === 3) {
+                    // No change, original query has all nececssary fields
```
Comment (sahiga) [inline src/features/metadata-based-view/__tests__/MetadataQueryAPIHelper.test.js:467]: Minor: typo in "necessary"

**Pull request merged**